index.js

```diff
@@ -262,8 +271,6 @@ exports.compileEVM = function (evmCode, stackTrace) {
           i = evmCode.length
         }
         break
-      case 'INVALID':
```

Comment:
so this is valid EVM code that has in valid opcode  `PUSH, 0x04, JUMP,
INVALID, JUMPDEST `. The ewasm out will have a (call $INVALID) which will
trap if its ever reached

On Sun, Aug 28, 2016 at 7:44 AM, Alex Beregszaszi <notifications@github.com>
wrote:

> In index.js
> <https://github.com/ewasm/evm2wasm/pull/92#discussion_r76530480>:
>
> > @@ -262,8 +271,6 @@ exports.compileEVM = function (evmCode, stackTrace) {
> >            i = evmCode.length
> >          }
> >          break
> > -      case 'INVALID':
>
> How would an invalid instruction get into the output?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ewasm/evm2wasm/pull/92/files/a9b2c4b78311bfe94a09b8062f5eb371b8b3742a#r76530480>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJqA6X0jlid6tyAgDfzbaQM61oySGzPks5qkXSCgaJpZM4JuztA>
> .
>